extensions/wikia/AssetsManager/config.php

```diff
@@ -913,6 +914,7 @@
 		'//extensions/wikia/AdEngine/js/template/interstitial.js',
 		'//extensions/wikia/AdEngine/js/template/playwire.js',
 		'//extensions/wikia/AdEngine/js/template/porvata.js',
+		'//extensions/wikia/AdEngine/js/template/roadblock.js',
```

Comment:
I believe we're not going to enable Roadblock on mobile. Can we handle it somehow?

Reply:
we want to display TLB only on mobile